components/org.wso2.carbon.identity.auth.valve/src/main/java/org/wso2/carbon/identity/auth/valve/util/APIErrorResponseHandler.java

```diff
@@ -39,6 +43,8 @@ public class APIErrorResponseHandler {
     private static final Log log = LogFactory.getLog(APIErrorResponseHandler.class);
 
     private static final String AUTH_HEADER_NAME = "WWW-Authenticate";
+    private static final String AUTH_FAILURE_REASON = "Auth-Failure-Reason";
+    private static final String AUTH_FAILURE_REASON_ACCOUNT_LOCKED = "account locked";
```

Comment:
```suggestion
    private static final String AUTH_FAILURE_REASON_ACCOUNT_LOCKED = "Account locked";
```

Reply:
Have used the prevalent format, commencing with a lowercase letter, similar to the standard convention. For instance, refer to this link: https://github.com/wso2-extensions/identity-carbon-auth-rest/blob/385b7d90b686aba21744b0ea63ed054080497806/components/org.wso2.carbon.identity.auth.valve/src/main/java/org/wso2/carbon/identity/auth/valve/util/APIErrorResponseHandler.java#L66C13-L66C42.